### DISCUSSION TITLE
Does this repository have a reason to not provide requirements.txt?

### DISCUSSION THREAD

**Discussion by ag0n1k** [koxudaxi/datamodel-code-generator#581] [Q&A] [answered]
I came up to understand the new way to provide extra.allow into my model to update the library from older version, and suddenly I have to run my tests about 10 times to know the full list of modules that this library uses, here are the list:
```
argcomplete==1.12.3
attrs==21.2.0
black==21.9b0
click==8.0.3
freezegun==1.1.0
inflect==5.3.0
iniconfig==1.1.1
isort==5.9.3
Jinja2==3.0.2
MarkupSafe==2.0.1
mypy-extensions==0.4.3
packaging==21.0
pathspec==0.9.0
platformdirs==2.4.0
pluggy==1.0.0
py==1.10.0
pydantic==1.8.2
pyparsing==2.4.7
PySnooper==1.0.0
pytest==6.2.5
python-dateutil==2.8.2
PyYAML==6.0
regex==2021.10.8
six==1.16.0
toml==0.10.2
tomli==1.2.1
typing-extensions==3.10.0.2
```

Is there a reason why there is no such information at repository?

**Comment by koxudaxi** [comment, marked as answer]
@ag0n1k 
We switched to poetry. we can check [poetry.lock](https://github.com/koxudaxi/datamodel-code-generator/blob/master/poetry.lock)
Does the file resolve your problem?
